specs/chains/chains.spec

```diff
@@ -0,0 +1,40 @@
+PIPELINES-27
+# Tekton Chains tests
+
+Precondition:
+* Validate Operator should be installed
+
+## Using Tekton Chains to create and verify task run signatures: PIPELINES-27-TC01
+Tags: chains, e2e, taskrun, sanity
+Component: Chains
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps: 
+    * Update the TektonConfig with taskrun format as "in-toto" taskrun storage as "tekton" oci storage as "" transparency mode as "false"
```

Comment:
@VeereshAradhya what do you think about this? It's following similar mechanism as pruner and addon tests but it's it too long? Shouldn't we split it into multiple steps - e.g. one for format, one for storage and one for transparency?